scipy/signal/filter_design.py

```diff
@@ -34,18 +34,22 @@ class BadCoefficients(UserWarning):
 abs = absolute
 
 
-def findfreqs(num, den, N):
+def findfreqs(num, den, N, input='ba'):
```

Comment:
I agree, but its maybe outside of the scope of this PR. I think it needs to happen in a unified way when we make this kind of change across the board.